lib/src/skopeo.rs

```diff
@@ -0,0 +1,23 @@
+use anyhow::Result;
+use serde::Deserialize;
+use std::process::Command;
+
+#[derive(Deserialize)]
+#[serde(rename_all = "PascalCase")]
+pub(crate) struct Inspect {
+    pub(crate) digest: String,
+}
+
+/// Given an image ID, return its manifest digest
+pub(crate) fn oci_archive_digest(imageref: &str) -> Result<String> {
```

Comment:
This isn't specific to oci archives right?  If we want it to be, let's make this a `&Utf8Path`, and then explicitly specify `dir:` below say.

If we want to keep it general...we also already use https://docs.rs/containers-image-proxy/latest/containers_image_proxy/ which is a handy Rust client for talking to skopeo, but I don't really in the end have a problem with being practical and just forking it either.

Reply:
(Forgot to mention, see below where we could also probably drop this code)

Reply:
Dropped (see below).